### DISCUSSION TITLE
bind Qtable property on anonther Qtable property

### DISCUSSION THREAD

**Discussion by D042657** [quasarframework/quasar#14571] [General - Components / Directives / etc]
Hello there,

I would like to switch dense mode depending on whether qtable is in fullscreen. I tried the line below but it doesn't work.

:dense="fullscreen ? true : false"

Thx for any advice.

BR
Heiko

**Comment by MilosPaunovic** [comment]
This should work if you're setting the `fullscreen` property properly, so without a code reproduction there isn't much help you could get.

**Reply by D042657** [reply]
Hi Milos,

thx for the swift reply. Please see the attached code below and a screenshot of the error Visual Code is reporting.

[component.txt](https://github.com/quasarframework/quasar/files/9725118/component.txt)


![image](https://user-images.githubusercontent.com/32865247/194320950-c9ca30e8-9972-4a0e-8b37-2b50ae80b012.png)

**Reply by MilosPaunovic** [reply]
What is the `fullscreen` in your code? Provide a Codepen reproduction of the issue.
